client/components/themes-list/test/index.jsx

```diff
@@ -1,39 +1,34 @@
 /**
  * External dependencies
  */
-var assert = require( 'chai' ).assert,
-	React = require( 'react' ),
-	TestUtils = require( 'react-addons-test-utils' ),
-	mockery = require( 'mockery' ),
-	sinon = require( 'sinon' );
-
-function mockComponent( displayName ) {
-	return React.createClass( {
-		displayName,
-		render: () => { return <div/> }
-	} );
-};
+import { assert } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import useMockery from 'test/helpers/use-mockery';
+import { useSandbox } from 'test/helpers/use-sinon';
 
 describe( 'ThemesList', function() {
+	let React, TestUtils, ThemesList;
+	useSandbox();
 	this.timeout( 10 * 1000 );
```

Comment:
can probably drop the timeout too, we globally timeout at 10 seconds under circle now.